docs/site/pages/plugins/asset-provider.mdx

```diff
@@ -9,7 +9,61 @@ The Asset Provider Plugin enables users to easily register UI components to rend
 
 ### Usage
 
+Use the Asset Provider plugin the register your custom assets. In this example, we have components that are rendering a customer asset from type `custom-asset` and `custom`.
```

Comment:
A brief expiation of what the below code blocks symbolize might be a helpful.

Reply:
Also organizationally it might make more sense if its after the initialization/setup of the plugin.